Formula/g/gnu-cobol.rb

```diff
@@ -1,7 +1,8 @@
 class GnuCobol < Formula
-  desc "Implements much of the COBOL 85 and COBOL 2002 standards"
-  homepage "https://sourceforge.net/projects/gnucobol/"
-  url "https://downloads.sourceforge.net/project/gnucobol/gnucobol/3.2/gnucobol-3.2.tar.xz"
+  desc "COBOL85-202x compiler supporting lots of dialect specific extensions"
+  homepage "https://www.gnu.org/software/gnucobol/"
```

Comment:
All links on there still refer to sourceforge. How is this different?

Reply:
This page is the official one and will probable point to something else in the future. It won't go away and most important: is under actual control of the GnuCOBOL team, while SF is under control of its owner.